packages/test/test-end-to-end-tests/src/test/gc/gcTombstoneDataStores.spec.ts

```diff
@@ -757,57 +771,105 @@ describeNoCompat("GC data store tombstone tests", (getTestObjectProvider) => {
-			// Logs a tombstone and sweep ready error
-			let dataObject: ITestDataObject;
-			await assert.doesNotReject(async () => {
-				dataObject = await requestFluidObject<ITestDataObject>(container, unreferencedId);
-			}, `Should be able to request a tombstoned datastore.`);
-			// Modifying the tombstoned datastore should not fail since ThrowOnTombstoneUsage is not enabled.
-			// Logs a submitMessage error
-			assert.doesNotThrow(
-				() => dataObject._root.set("send", "op"),
-				`Should be able to send ops for a tombstoned datastore.`,
-			);
+		});
 
-			// Wait for the above op to be process. That will result in another error logged during process.
-			// Both the summarizing container and the submitting container log a process error
-			await provider.ensureSynchronized();
-		},
-	);
+		itExpects(
+			"Load via Request and Modify",
+			[
+				{
+					eventName: "fluid:telemetry:ContainerRuntime:GC_Tombstone_DataStore_Requested",
+					clientType: "interactive",
+				},
+				{
+					eventName:
+						"fluid:telemetry:FluidDataStoreContext:GC_Tombstone_DataStore_Changed",
+					callSite: "submitMessage",
+					clientType: "interactive",
+				},
+				{
+					eventName:
+						"fluid:telemetry:FluidDataStoreContext:GC_Tombstone_DataStore_Changed",
+					callSite: "process",
+					clientType: "noninteractive/summarizer",
+				},
+				{
+					eventName:
+						"fluid:telemetry:FluidDataStoreContext:GC_Tombstone_DataStore_Changed",
+					callSite: "process",
+					clientType: "interactive",
+				},
+			],
+			async () => {
+				const { unreferencedId, summarizingContainer, summarizer } =
+					await summarizationWithUnreferencedDataStoreAfterTime(sweepTimeoutMs);
+				await sendOpToUpdateSummaryTimestampToNow(summarizingContainer);
+
+				// The datastore should be tombstoned now
+				const { summaryVersion } = await summarize(summarizer);
+				const container = await loadContainer(summaryVersion);
+				// Requesting the tombstoned data store should succeed since ThrowOnTombstoneLoad is not enabled.
+				// Logs a tombstone and sweep ready error
+				let dataObject: ITestDataObject;
+				await assert.doesNotReject(async () => {
+					dataObject = await requestFluidObject<ITestDataObject>(
+						container,
+						unreferencedId,
+					);
+				}, `Should be able to request a tombstoned datastore.`);
+				// Modifying the tombstoned datastore should not fail since ThrowOnTombstoneUsage is not enabled.
+				// Logs a submitMessage error
+				assert.doesNotThrow(
+					() => dataObject._root.set("send", "op"),
+					`Should be able to send ops for a tombstoned datastore.`,
+				);
+
+				// Wait for the above op to be process. That will result in another error logged during process.
+				// Both the summarizing container and the submitting container log a process error
+				await provider.ensureSynchronized();
+			},
+		);
+
+		//* ONLY ONLY ONLY ONLY ONLY ONLY ONLY ONLY ONLY ONLY ONLY ONLY ONLY ONLY ONLY
+		itExpects.only(
```

Comment:
Can you add a test for "unreferenced' and "inactive" data stores as well to test that this will work with non manufactured handles as well. Plus we should have tests for all states and unreferenced time returned.